.github/workflows/docker.yml

```diff
@@ -23,7 +25,15 @@ jobs:
             --build-arg BUILDKIT_INLINE_CACHE=1 \
             --cache-from "dependabot/dependabot-core:latest" \
             .
-      - name: Push image to packages
+      - name: Push image to packages (latest)
+        if: '!contains(github.ref, "refs/tags")'
```

Comment:
I'm claiming this as an independent invention, but https://github.com/search?q=%22if%3A+contains%28github.ref%2C+%22tags%22%29%22&type=Code makes it seem ~known.

Maybe `refs/tags` is overkill given the masses' wisdom...